components/brave_new_tab_ui/containers/newTab/settings/assets/hide-icon.tsx

```diff
@@ -8,7 +8,7 @@ export default class HideIcon extends React.PureComponent {
 
   render () {
     return (
-      <svg width={'19'} height={'17'} xmlns={'http://www.w3.org/2000/svg'}>
+      <svg width={'100%'} height={'100%'} xmlns={'http://www.w3.org/2000/svg'}>
```

Comment:
nit: this should now have a viewBox as 19x17 is the space that the `<path>` coordinates are in. I think it would be `viewBox="0 0 19 17"`. Only a nit since you are explicitly setting the width and height to be what it's expecting in css, but if we scale this up or down in css it's going to have weird results AFAIK.

Reply:
That was an oversight, added 👍